### PR TITLE
improvement: showCommonExtensions support for OAS3 parameters

### LINKED ISSUE #5016: OAI 3's minimum/maximum validation doesn't show on UI
### Q&A
 - OS: Linux
 - Browser: Chrome
 - Version: 70
 - Method of installation: npm
 - Swagger-UI version: 3.18.3
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration

Example Swagger/OpenAPI definition:
```json
{
  "openapi": "3.0.0",
  "info": {
    "title": "Test API",
    "version": "1"
  },
  "paths": {
    "/test": {
      "get": {
        "parameters": [
         {
           "in": "query",
            "name": "test",
            "schema": {
            "type": "integer",
            "minimum": 1,
            "maximum": 10
          },
          "required": false
         }
        ],
        "responses": {
          "204": {"description": "success"}
        }
      }
    }
  }
}
```

Swagger-UI configuration options:
```js
SwaggerUI({
  // default config
})
```

### Describe the bug you're encountering

Minimum/maximum information doesn't show in parameter validation. In "Try it Out", if I put something that out of the range, even though the Execute button stop running, there is no red border on the input to indicate error.

May be related to https://github.com/swagger-api/swagger-ui/issues/4581

### To reproduce...

1. Go to https://editor.swagger.io/
2. Use example definition above
3. Click "Try it out"
4. Put 10000 for `test`
5. Click Execute

### Expected behavior

There should be red border to indicate error on `test` input.
There should be minimum, maximum validation information in UI. (Like it was for Swagger 2.0)

### Screenshots

screenshot_20181109_083230

### PR DESCRIPTION
**Pull request by hkosova**
### Description
Swagger UI has the [`showCommonExtensions`](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md#user-content-showcommonextensions) option that displays the `maximum`, `minimum`, `maxLength`, `minLength`, and `pattern` attributes for primitive parameters in OAS2 definitions. This PR makes this work for OAS3 parameters as well.

### Motivation and Context
Fixes #5016.

### How Has This Been Tested?
Manually.

### Screenshots (if appropriate):
Here, Swagger UI is configured with `showCommonExtensions: true`.

Swagger UI showing parameter minimum, maximum, minLength, maxLength, and pattern

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [x] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [x] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/core/components/parameter-row.jsx [hunk 1]
```diff
@@ -220,7 +220,7 @@ export default class ParameterRow extends Component {
     let itemType = schema.getIn(["items", "type"])
 
     let value = paramWithMeta ? paramWithMeta.get("value") : ""
-    let commonExt = showCommonExtensions ? getCommonExtensions(param) : null
+    let commonExt = showCommonExtensions ? getCommonExtensions(schema) : null
```
Comment (shockey) [inline src/core/components/parameter-row.jsx]: great reuse of our existing version-specific logic!
